lib/CL/devices/level0/npu/include/ze_command_queue_npu_ext.h

```diff
@@ -0,0 +1,84 @@
+/*
```

Comment:
Why do we copy these headers? Please add a link to the source somewhere?

Reply:
I think these are not distributed via the base level0 package. Added link to the origin (I think they are from) into the copied headers.

Reply:
not distributed in base, and also AFAICT there are no github or debian packages.